File: Python PowerPoint Examples/GeneralApiDemo/convertInRequest.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudpowerpoint.api.general_api.GeneralApi(configuration)

format = "Odp"  # Supported formats:Pptx/Ppsx/Ppt/Pps/Odp/Pdf/Xps/Ps/Pcl
file = "D:\\input\\Convert.pptx"
password = ""

result = api.convert_in_request(format=format, file=file, password=password)

File: PHP PowerPoint Examples/GeneralApiDemo/convertInRequest.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\GeneralApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new GeneralApi($configuration);

$format = "Pptx"; //Available format: Pptx, Ppt, Odp, Pps, Ppsx, Pdf, Xps, Ps, Pcl
$password = null;
$filename = "D:/data/ConvertInRequest.pptx";
$document = new SplFileObject($filename);
$result = $apiInstance->convertInRequest($format, $document, $password);
print_r($result);